yt/fields/field_info_container.py

```diff
@@ -398,8 +398,8 @@ def check_derived_fields(self, fields_to_check = None):
         unavailable = []
         fields_to_check = fields_to_check or list(self.keys())
         for field in fields_to_check:
-            mylog.debug("Checking %s", field)
-            if field not in self: raise RuntimeError
+            if field not in self:
+                raise RuntimeError("Could not find field %s" % str(field))
```

Comment:
Since you're fixing this, could you make this a `YTFieldNotFound` exception for consistency?
```suggestion
                raise YTFieldNotFound(str(field))
```

Reply:
Makes a lot of sense, should I thought of this... done ! Thanks !